SharpPcap/TimestampSourceResolution.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+namespace SharpPcap
+{
+    public enum TimestampSourceResolution
+    {
+        Low,
+        Medium,
```

Comment:
```suggestion
        Unknown,
```

Reply:
`Medium` timestamp precision does not exist

Reply:
You are correct. I thought I saw it in a header file but I likely just made it up. hah Fixed